.github/CODEOWNERS

```diff
@@ -1,18 +1 @@
-# The SVM team is in the process of migrating these subdirectories to a new
-# repo and would like to avoid introducing dependencies in the meantime.
-/builtins-default-costs/ @anza-xyz/svm
-/compute-budget/ @anza-xyz/svm
-/compute-budget-instruction/ @anza-xyz/svm @anza-xyz/fees
-/fee/ @anza-xyz/fees
-/log-collector/ @anza-xyz/svm
-/program-runtime/ @anza-xyz/svm
-/programs/bpf_loader/ @anza-xyz/svm
-/programs/compute-budget/ @anza-xyz/svm
-/programs/loader-v4/ @anza-xyz/svm
-/runtime-transaction/ @anza-xyz/tx-metadata
-/svm-conformance/ @anza-xyz/svm
-/svm-rent-collector/ @anza-xyz/svm
-/svm-transaction/ @anza-xyz/svm
-/svm/ @anza-xyz/svm
-/svm/examples/Cargo.lock
-/transaction-view/ @anza-xyz/tx-metadata
```

Comment:
Ideally  for these paths the backport SME reviewer(s) would be a member of the appropriate groups. We'll need to monitor that socially for now, since GH doesn't support requiring a  review from each of multiple groups